Sync `cssom-view` test from WPT upstream

Ahmad-S792 opened the pull request:
#### bdca5569edf54e2e43a8acfe4656a8c884b2b47f
<pre>
Sync `cssom-view` test from WPT upstream

<a href="https://bugs.webkit.org/show_bug.cgi?id=279292">https://bugs.webkit.org/show_bug.cgi?id=279292</a>
<a href="https://rdar.apple.com/135448593">rdar://135448593</a>

Reviewed by Tim Nguyen.

Upstream commit: <a href="https://github.com/web-platform-tests/wpt/commit/ff8ec78f3504a09f25b9a83fdc42b0db94332605">https://github.com/web-platform-tests/wpt/commit/ff8ec78f3504a09f25b9a83fdc42b0db94332605</a>

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/HTMLBody-ScrollArea_quirksmode.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/elementScroll-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/negativeMargins-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/negativeMargins.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/offsetParent_element_test.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/resize-event-on-initial-layout-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/resize-event-on-initial-layout.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/resources/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scroll-behavior-default-css-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scroll-behavior-default-css.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scroll-behavior-element-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scroll-behavior-element.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scroll-behavior-main-frame-root-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scroll-behavior-main-frame-root.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scroll-behavior-main-frame-window-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scroll-behavior-main-frame-window.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scroll-behavior-scrollintoview-nested-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scroll-behavior-scrollintoview-nested.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scroll-behavior-smooth-navigation.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scroll-behavior-smooth-positions-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scroll-behavior-smooth-positions.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scroll-behavior-smooth.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scroll-behavior-subframe-root-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scroll-behavior-subframe-root.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scroll-behavior-subframe-window-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scroll-behavior-subframe-window.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollIntoView-horizontal-tb-writing-mode-and-rtl-direction.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollIntoView-horizontal-tb-writing-mode.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollIntoView-scrollMargin.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollIntoView-scrollPadding.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollIntoView-sideways-lr-writing-mode-and-rtl-direction.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollIntoView-sideways-lr-writing-mode.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollIntoView-sideways-rl-writing-mode-and-rtl-direction.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollIntoView-sideways-rl-writing-mode.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollIntoView-smooth-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollIntoView-smooth.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollIntoView-vertical-lr-writing-mode-and-rtl-direction.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollIntoView-vertical-lr-writing-mode.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollIntoView-vertical-rl-writing-mode.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollLeftTop.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/support/scroll-behavior.js:
(observeScrolling):
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/client-props-zoom-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/client-props-zoom.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/Element-currentCSSZoom-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/Element-currentCSSZoom.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/getBoundingClientRect-zoom-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/getBoundingClientRect-zoom.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/getClientRects-zoom-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/getClientRects-zoom.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/image-x-y-zoom-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/image-x-y-zoom.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/offsetTop-offsetLeft-with-zoom-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/offsetTop-offsetLeft-with-zoom.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/offsetTopLeft-table-caption-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/offsetTopLeft-table-caption.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/overflow-hidden-smooth-scroll-crash.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/range-bounding-client-rect-with-nested-text-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/range-bounding-client-rect-with-nested-text.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/resize-event-on-initial-layout-001-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/resize-event-on-initial-layout-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/resize-event-on-initial-layout-002-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/resize-event-on-initial-layout-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/resources/simultaneousScrollIntoViews.js:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scroll-zoom-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scroll-zoom.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollIntoView-align-scrollport-covering-child-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollIntoView-align-scrollport-covering-child.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollIntoView-multiple-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollIntoView-multiple-nested-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollIntoView-multiple-nested.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollIntoView-multiple.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollWidthHeight-negative-margin-001-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollWidthHeight-negative-margin-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollWidthHeight-negative-margin-002-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollWidthHeight-negative-margin-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/smooth-scroll-in-load-event-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/smooth-scroll-in-load-event.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/smooth-scroll-nonstop-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/smooth-scroll-nonstop.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/smooth-scrollIntoView-with-smooth-fragment-scroll-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/smooth-scrollIntoView-with-smooth-fragment-scroll-iframe.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/smooth-scrollIntoView-with-smooth-fragment-scroll.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/smooth-scrollIntoView-with-unrelated-gesture-scroll-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/smooth-scrollIntoView-with-unrelated-gesture-scroll.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/subpixel-sizes-and-offsets.tentative-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/subpixel-sizes-and-offsets.tentative.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/WEB_FEATURES.yml:
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/css/cssom-view/smooth-scroll-in-load-event-expected.txt:
* LayoutTests/platform/mac-wk1/TestExpectations:

Canonical link: <a href="https://commits.webkit.org/283304@main">https://commits.webkit.org/283304@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/dea772b8a45510c1ced5d4f38d725aafe503f771

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/65910 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/45283 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/18529 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/69939 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/59/builds/16519 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/53082 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/16801 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/52893 "Passed tests") | [✅ 🧪 wincairo-tests](https://ews-build.webkit.org/#/builders/60/builds/11476 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/68977 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/41793 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/57058 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/33528 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/38464 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/14435 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/15395 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/60311 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/14781 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/71642 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/87/builds/9865 "Built successfully") | [❌ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/62/builds/14187 "Found 1 new test failure: http/wpt/mediastream/mediastreamtrackprocessor-videoframe-timestamp.html (failure)") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/60208 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/86/builds/9897 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/57123 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/60498 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/14523 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/88/builds/8144 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/119/builds/1782 "Passed tests") | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/41091 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/42167 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/43350 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/41911 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

nt1m reviewed (approved):
Please make sure everything is gardened correctly before landing

Ahmad-S792 committed:
Sync `cssom-view` test from WPT upstream

https://bugs.webkit.org/show_bug.cgi?id=279292
rdar://135448593

Reviewed by Tim Nguyen.

Upstream commit: https://github.com/web-platform-tests/wpt/commit/ff8ec78f3504a09f25b9a83fdc42b0db94332605

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/HTMLBody-ScrollArea_quirksmode.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/elementScroll-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/negativeMargins-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/negativeMargins.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/offsetParent_element_test.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/resize-event-on-initial-layout-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/resize-event-on-initial-layout.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/resources/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scroll-behavior-default-css-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scroll-behavior-default-css.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scroll-behavior-element-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scroll-behavior-element.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scroll-behavior-main-frame-root-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scroll-behavior-main-frame-root.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scroll-behavior-main-frame-window-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scroll-behavior-main-frame-window.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scroll-behavior-scrollintoview-nested-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scroll-behavior-scrollintoview-nested.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scroll-behavior-smooth-navigation.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scroll-behavior-smooth-positions-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scroll-behavior-smooth-positions.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scroll-behavior-smooth.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scroll-behavior-subframe-root-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scroll-behavior-subframe-root.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scroll-behavior-subframe-window-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scroll-behavior-subframe-window.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollIntoView-horizontal-tb-writing-mode-and-rtl-direction.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollIntoView-horizontal-tb-writing-mode.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollIntoView-scrollMargin.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollIntoView-scrollPadding.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollIntoView-sideways-lr-writing-mode-and-rtl-direction.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollIntoView-sideways-lr-writing-mode.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollIntoView-sideways-rl-writing-mode-and-rtl-direction.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollIntoView-sideways-rl-writing-mode.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollIntoView-smooth-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollIntoView-smooth.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollIntoView-vertical-lr-writing-mode-and-rtl-direction.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollIntoView-vertical-lr-writing-mode.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollIntoView-vertical-rl-writing-mode.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollLeftTop.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/support/scroll-behavior.js:
(observeScrolling):
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/client-props-zoom-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/client-props-zoom.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/Element-currentCSSZoom-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/Element-currentCSSZoom.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/getBoundingClientRect-zoom-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/getBoundingClientRect-zoom.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/getClientRects-zoom-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/getClientRects-zoom.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/image-x-y-zoom-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/image-x-y-zoom.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/offsetTop-offsetLeft-with-zoom-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/offsetTop-offsetLeft-with-zoom.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/offsetTopLeft-table-caption-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/offsetTopLeft-table-caption.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/overflow-hidden-smooth-scroll-crash.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/range-bounding-client-rect-with-nested-text-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/range-bounding-client-rect-with-nested-text.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/resize-event-on-initial-layout-001-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/resize-event-on-initial-layout-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/resize-event-on-initial-layout-002-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/resize-event-on-initial-layout-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/resources/simultaneousScrollIntoViews.js:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scroll-zoom-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scroll-zoom.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollIntoView-align-scrollport-covering-child-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollIntoView-align-scrollport-covering-child.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollIntoView-multiple-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollIntoView-multiple-nested-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollIntoView-multiple-nested.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollIntoView-multiple.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollWidthHeight-negative-margin-001-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollWidthHeight-negative-margin-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollWidthHeight-negative-margin-002-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/scrollWidthHeight-negative-margin-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/smooth-scroll-in-load-event-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/smooth-scroll-in-load-event.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/smooth-scroll-nonstop-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/smooth-scroll-nonstop.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/smooth-scrollIntoView-with-smooth-fragment-scroll-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/smooth-scrollIntoView-with-smooth-fragment-scroll-iframe.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/smooth-scrollIntoView-with-smooth-fragment-scroll.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/smooth-scrollIntoView-with-unrelated-gesture-scroll-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/smooth-scrollIntoView-with-unrelated-gesture-scroll.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/subpixel-sizes-and-offsets.tentative-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/subpixel-sizes-and-offsets.tentative.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/WEB_FEATURES.yml:
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/css/cssom-view/smooth-scroll-in-load-event-expected.txt:
* LayoutTests/platform/mac-wk1/TestExpectations:

Canonical link: https://commits.webkit.org/283304@main
